tests/run_live_k8s_tests.py

```diff
@@ -1,6 +1,5 @@
 import sys
 import os
-from fnmatch import fnmatch
 from time import time
 from ruamel.yaml import YAML
 import argparse
```

Comment:
```suggestion
import argparse
from ruamel.yaml import YAML
```
standard imports should come before 3rd-party imports